common/src/commonMain/kotlin/org/hisp/dhis/common/App.kt

```diff
@@ -129,6 +130,7 @@ fun Main() {
             Components.INPUT_INTEGER -> InputIntegerScreen()
             Components.INPUT_NUMBER -> InputNumberScreen()
             Components.INPUT_LETTER -> InputLetterScreen()
+            Components.AGE_FIELD -> AgeFieldScreen()
```

Comment:
Sott again, I think it should be called `InputAgeScreen` in alignement with the other public components

Reply:
done!